src/layman/layer/db/__init__.py

```diff
@@ -145,14 +159,15 @@ def check_new_layername(workspace, layername, conn_cur=None):
 
 
 def get_text_column_names(workspace, layername, conn_cur=None):
```

Comment:
Use table_name as optional parameter for this other methods in this file (if reasonable)